eosio/deployment/change_ownership.sh

```diff
@@ -6,7 +6,11 @@ pushd "$(dirname "${BASH_SOURCE[0]}")"
 # Change the private key that controls a role in an account by giving a new public key.
```

Comment:
Comment is no longer accurate. It's not "a role", it's the owner and active roles of a very specific eos account.